ndt7/model/measurement.go

```diff
@@ -8,4 +8,5 @@ type Measurement struct {
 	ConnectionInfo *ConnectionInfo `json:",omitempty" bigquery:"-"`
 	BBRInfo        *BBRInfo        `json:",omitempty"`
 	TCPInfo        *TCPInfo        `json:",omitempty"`
+	WSInfo         *WSInfo	       `json:",omitempty"`
```

Comment:
@bassosimone, you've said at #232 that it should rather be `PingInfo`. I'd suggest `WSPingInfo` then to avoid possible confusion about different pings at different network layers. Does it sound okay?

Websocket ping-pong may be not end-to-end as well in some weird hypothetical cases involving frame-based websocket load balancer. I consider those cases extremely unlikely for `wss` but `ws` has some chances of alike network interference.

Reply:
Yes, the name you propose sounds better